README.md

```diff
@@ -7,7 +7,7 @@ The Datadog Agent Ansible role installs and configures the Datadog Agent and int
 The Datadog Agent Ansible role is available through 2 different channels:
 
 * As part of the Datadog collection, accessible under the [datadog.dd](https://galaxy.ansible.com/ui/repo/published/datadog/dd/) name on Ansible Galaxy (recommended).
-* As a standalone role, accessible under the [datadog.datadog](https://galaxy.ansible.com/ui/repo/published/datadog/dd/content/role/agent/) name on Ansible Galaxy (legacy).
+* As a standalone role, accessible under the [datadog.datadog](https://galaxy.ansible.com/ui/standalone/roles/DataDog/datadog/) name on Ansible Galaxy (legacy).
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏We could reuse the existing link here
```suggestion
* As a standalone role, accessible under the [datadog.datadog][1] name on Ansible Galaxy (legacy).
```